site/assets/js/tac.js

```diff
@@ -31,8 +33,9 @@
 
     document.getElementById('tarteaucitronCloseAlert').classList.add('btn', 'btn-sm', 'ms-lg-2')
     alert.querySelector('.tarteaucitronAllow').classList.add('btn', 'btn-sm', 'btn-success', 'mx-sm-2', 'ms-lg-auto', 'my-2', 'my-lg-0')
+    alert.querySelector('.tarteaucitronAllow').innerHTML = tarteaucitron.lang.acceptAll
```

Comment:
Is the key to use `acceptAll` ("OK, tout accepter") or `allowAll` ("Autoriser tous les cookies") ?
In the source code of tarteaucitron it is the `allowAll` key which seems to be used.